### PR TITLE
Update Java configuration for Log Analytics

### PR DESCRIPTION
**Pull request by alexeldeib**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/operationalinsights/data-plane/readme.md [hunk 1]
```diff
@@ -134,14 +134,13 @@ output-folder: $(go-sdk-folder)/services/operationalinsights/v1/operationalinsig
...
 
 These settings apply only when `--java` is specified on the command line.
-Please also specify `--azure-libraries-for-java-folder=<path to the root directory of your azure-libraries-for-java clone>`.
 
 ``` yaml $(java)
 java:
   azure-arm: true
-  fluent: true
-  namespace: com.microsoft.azure.operationalinsights
+  fluent: false
```
Comment (alexeldeib) [inline specification/operationalinsights/data-plane/readme.md]: There is no existing client that uses this generation.

File: specification/operationalinsights/data-plane/readme.md [hunk 2]
```diff
@@ -134,14 +134,13 @@ output-folder: $(go-sdk-folder)/services/operationalinsights/v1/operationalinsig
 ## Java
 
 These settings apply only when `--java` is specified on the command line.
-Please also specify `--azure-libraries-for-java-folder=<path to the root directory of your azure-libraries-for-java clone>`.
 
 ``` yaml $(java)
 java:
```
Comment (alexeldeib) [inline specification/operationalinsights/data-plane/readme.md]: There is no existing client that uses the old configuration.

File: specification/operationalinsights/data-plane/readme.md [hunk 3]
```diff
@@ -25,7 +25,7 @@ These are the global settings for the OperationalInsightsData API.
 
 ``` yaml
 title: OperationalInsightsDataClient
-description: Operational Insights Data Client
+description: Log Analytics Data Plane Client
```
Comment (alexeldeib) [inline specification/operationalinsights/data-plane/readme.md]: Reflecting branding, which has been different for some time (keeping title due to existing packages dependent on it; will override on a per package basis for new releases.

**Comment by alexeldeib** [comment]
Related to https://github.com/Azure/azure-sdk-for-java/pull/2113

**Comment by marstr** [comment]
I'd like @jianghaolu to validate the java configuration changes. Other than that, this looks good.

File: specification/operationalinsights/data-plane/readme.md [hunk 4]
```diff
@@ -134,14 +134,13 @@ output-folder: $(go-sdk-folder)/services/operationalinsights/v1/operationalinsig
 ## Java
 
 These settings apply only when `--java` is specified on the command line.
-Please also specify `--azure-libraries-for-java-folder=<path to the root directory of your azure-libraries-for-java clone>`.
```
Comment (jianghaolu) [inline specification/operationalinsights/data-plane/readme.md]: Why do you remove this?

**Comment by marstr** [comment]
@jianghaolu, does the update look good to you?

**Comment by jianghaolu** [comment]
@marstr Yes! Please feel free to merge.

**Review by marstr — APPROVED** [review]

**Pull request merged**